site/app/templates/admin/users/UserForm.twig

```diff
@@ -94,6 +94,14 @@
                 <option value="4">Student</option>
             </select>
         </label>
+        <label for="registration_type" class="full-width">
+            Registration Type:
+            <select name="registration_type" id="registration_type">
+                <option value="graded">graded</option>
```

Comment:
Also nitpicky, but we could possibly refactor this like the other input options to use something like `{% for reg in reg_types %}`